### PR TITLE
Minor cleanup in http/server.d: remove constructor + add const

### PR DESCRIPTION
**Pull request by wilzbach**
A couple of small things that I found by making a short pass through `http/server.d`, btw I am always positively surprised how easy Vibe.d's code is to read :+1:

### CODE REVIEW

File: http/vibe/http/server.d [hunk 1]
```diff
@@ -624,15 +624,7 @@ final class HTTPServerSettings {
 
 		The default value is 60 seconds; set to Duration.zero to disable pings.
 	*/
-	Duration webSocketPingInterval;// = dur!"seconds"(60);
-
-	this()
-	@safe {
-		// need to use the contructor because the Ubuntu 13.10 GDC cannot CTFE dur()
```
Comment (wilzbach) [inline http/vibe/http/server.d]: Since 0.7.29 `gdc` isn't officially supported anymore.

Comment (s-ludwig) [inline http/vibe/http/server.d]: Heh, who would have guessed that I had made the same changes yesterday :) https://github.com/rejectedsoftware/vibe.d/commit/42fdf87aad30442837da20b9eeef75c149adcb14

I'll merge this though and just drop that branch.

**Comment by wilzbach** [comment]
> Heh, who would have guessed that I had made the same changes yesterday :) 42fdf87
> I'll merge this though and just drop that branch.

Hehe. Sorry.

image

@s-ludwig I think you need to disable the "require branch to be up-to-date" - otherwise it's an rebase nightmare  (though the bot might [soon](https://github.com/dlang-bots/dlang-bot/pull/109) be able to rebase PRs automatically).

**Comment by s-ludwig** [comment]
> I think you need to disable the "require branch to be up-to-date" - otherwise it's an rebase nightmare (though the bot might soon be able to rebase PRs automatically).

Already done! BTW, thanks for the detailed guide! Everything should be in place now on this side.

**Pull request merged**